Manual_for_achitecture_browser/index.asciidoc

```diff
@@ -0,0 +1,62 @@
+= Manual for architecture browser
+These content help you to know how you can contribute to the content of the architecture browser.
+
+
+== Fetch upstream or fork repository
+If the repository already exists in your account, it will eventually become out of date, you have to make the changes that were made on the main repository. To do that, you have to click on "Fecht upstream " and choose "Fecht and merge".
+
+image::./addManual_image/fecht.png[fecht]
+But if it doesn't exist, then you have to fork it, so that the main repository will be generated in your own account. To do that, please click on the icon "Fork" from the main repository.
+
+image::./addManual_image/fork.png[Fork_Repository,100,100]
+
+== Open the repository on Gitpod
+The repository is already on your account. 
+Gitpod is already configured so that origin remote is not the main repository but your fork.
+You have only to go in the Readme.md and click on "https://gitpod.io/from-referrer/ ". This unique URL will automatically detect which repository it was visited from, and open that in Gitpod (using the Referrer HTTP header).
+
+
+== Create or Change tutorial
+You have the possibility to add a new tutorial by creating a new tutorial, you can also change the content of existing tutorial.
```

Comment:
You have the possibility to add a new tutorial by creating a new folder with an index.asciidoc or you can change the content of an existing tutorial.....

You have to make much clearer what the user has to do to create a new tutorial.